sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample2_DetectLastPointAnomaly.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Azure;
+using Azure.AI.AnomalyDetector;
+using Azure.AI.AnomalyDetector.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+class Sample2_DetectLastPointAnomaly
```

Comment:
As we moved these samples to the tests folder, can we remove it from here and create md files like [this](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md) and add the test file link at the bottom of the md file?

Reply:
Removed this file.

---
In reply to: [482527888](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r482527888) [](ancestors = 482527888)